src/BLSSignatureChecker.sol

```diff
@@ -73,6 +73,10 @@ contract BLSSignatureChecker is IBLSSignatureChecker {
         // loop through every quorumNumber and keep track of the apk
         BN254.G1Point memory apk = BN254.G1Point(0, 0);
         for (uint i = 0; i < quorumNumbers.length; i++) {
+            require(
+                stakeRegistry.quorumUpdateTimestamp(uint8(quorumNumbers[i])) + 1 weeks <= block.timestamp,
+                "BLSSignatureChecker.checkSignatures: StakeRegistry updates must be within a week"
+            );
```

Comment:
we had discussed that there should likely be a switch for enabling/disabling this check.

this seems correct as a first pass, but we should probably figure out where the switch lives + access control.

the `1 weeks` here is also a "magic number" -- it would be better practice to name this constant, I think. It also seems possible we want this to be a variable rather than a constant, so might be good to bring that up in discussion with the AVS team?

Reply:
https://github.com/Layr-Labs/eigenlayer-middleware/pull/62/commits/a4212dac9f000a0e73f2b99be7a69daef3520453
Updated it now to constant. Will bring this up later in discussion